content/Products/OpenshiftMonitoring/telemetry.md

```diff
@@ -0,0 +1,238 @@
+
+### Why is there a limit on the number of metrics that can be collected?
+
+See the previous section. Every metric shipped to Telemetry has to be multiplied by the number of connected clusters that may be sending that metric. Pushing too many metrics from a single cluster may cause service degradation and resource exhaustion on both the in-cluster monitoring stack and on the Telemetry server side.
+
+### Will Telemetry automatically collect alerts?
+
+Yes, the Telemeter client is already configured to collect and send firing alerts. On Telemetry side, the alerts can be queried using the `alerts` metric.
+
+### How do I ship metrics via Telemetry for older OCP releases?
+
+Once you have updated the `telemeter-client` configuration in the `master` branch, you can create backports to older OCP releases. The procedure follows the usual OCP backport process which involves creating bug tickets in the `OCPBUGS` project (preferably assigned to your component) and opening pull requests in openshift/cluster-monitoring-operator against the desired `release-4.x` branches.
+
+### How do I get access to Telemtry?
+
+Check https://help.datahub.redhat.com/docs/interacting-with-telemetry-data
+
+### How do I instrument my component for Prometheus?
+
+Please refer to the following links for more details:
+* [Metric and label naming](https://prometheus.io/docs/practices/naming/) (upstream Prometheus documentation).
+* [Instrumentation](https://prometheus.io/docs/practices/instrumentation/) (upstream Prometheus documentation).
+* [Instrumenting Kubernetes](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/instrumentation.md)
+
+You can also reach out to the OpenShift monitoring team for advice.
+
+### How does the in-cluster monitoring stack scrape metrics from my component?
+
+If your component's metrics aren't already collected by the in-cluster monitoring stack, you need to deploy at least one [ServiceMonitor](https://prometheus-operator.dev/docs/operator/api/#monitoring.coreos.com/v1.ServiceMonitor) or one [PodMonitor](https://prometheus-operator.dev/docs/operator/api/#monitoring.coreos.com/v1.PodMonitor) resource in your component's namespace.
+
+If your component is deployed by the Cluster Version Operator (CVO), it is enough to add the manifest to the CVO payload.
+
+Again you can reach out to the OpenShift monitoring team for advice.
+
+### How is the communication secured between the Telemeter client and server?
+
+The Telemetry client authenticates against the Telemeter server using the cluster's pull secret. The Telemeter server verifies that the pull secret is valid and matches with the cluster's identifier. The Telemetry protocol uses HTTPS for encryption.
+
+Finally the Telemeter server will only allow metrics which are explicitly allowed by its running configuraiton.
+
```

Comment:
Do you think it makes sense to add a link to the README of telemeter since it has a lot of info regarding this?
```suggestion
More info can be found https://github.com/openshift/telemeter#telemeter
```

Reply:
This is indeed good info but I'd rather see it linked from/included into https://rhobs-handbook.netlify.app/services/rhobs/use-cases/telemetry.md/?